target/snitch_cluster/README.md

```diff
@@ -104,6 +147,16 @@ The previous commands will run the simulation in your current terminal. You can
 questa-2022.3 bin/snitch_cluster.vsim.gui sw/apps/blas/axpy/build/axpy.elf
 ```
 
+You can also produce a `vcd` file which you can display on [gtkwave](https://gtkwave.sourceforge.net/).
+
+```shell
+# Add the --vcd at the end to generate vcd files. This produces sim.vcd
+bin/snitch_cluster.vlt sw/apps/blas/axpy/build/axpy.elf --vcd
+
+# Display the vcd file in gtkwave
+gtkwave sim.vcd
+```
+
 For Banshee, you need to give a specific cluster configuration to the simulator with the flag `--configuration <cluster_config.yaml>`. A default Snitch cluster configuration is given (`src/banshee.yaml`). The flag `--trace` enables the printing of the traces similar to the RTL simulation.
```

Comment:
Our software is not tested on banshee, maybe add a warning here?

Reply:
OK!